konfera/register/views.py

```diff
@@ -4,8 +4,37 @@
 
 from konfera.models.event import Event
 from konfera.models.ticket import REQUESTED
-from konfera.models.ticket_type import TicketType, VOLUNTEER, PUBLIC
-from konfera.register.forms import VolunteerRegistrationForm
+from konfera.models.ticket_type import TicketType, VOLUNTEER, PUBLIC, PRIVATE, ACTIVE
+from konfera.register.forms import RegistrationForm
+
+
+def private_registration(request, event_slug, ticket_uuid):
+    context = dict()
+    event = get_object_or_404(Event, slug=event_slug)
+    ticket_type = get_object_or_404(TicketType, event=event.id, uuid=ticket_uuid, accessibility=PRIVATE)
+    if ticket_type.status != ACTIVE:
+        messages.error(request, "This ticket type is not available")
```

Comment:
The strings in the class where not translated, so I assumed it was just english.

Added the translations...